client/components/ManageTestQueue/ManageTestQueue.module.css

```diff
@@ -0,0 +1,103 @@
+.manage-disclosure-container {
+  > span {
+    display: block;
+    margin-bottom: 1rem;
+  }
+
+  /* Add Test Plan to Test Queue button */
+  > button {
+    padding: 0.5rem 1rem;
+    margin-top: 1rem;
+  }
+
+  .disclosure-row-manage-ats {
+    display: grid;
+    grid-auto-flow: column;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+    grid-gap: 1rem;
+
+    .ats-container {
+      grid-column: 1 / span 2;
+    }
+
+    .at-versions-container {
+      display: flex;
+      flex-direction: column;
+      grid-column: 3 / span 3;
+    }
+
+    .disclosure-buttons-row {
+      display: flex;
+      flex-direction: row;
+      justify-content: flex-start;
+
+      > button {
+        margin: 0;
+        padding: 0;
+        color: #275caa;
+        border: none;
+        background-color: transparent;
+
+        &:nth-of-type(2) {
+          margin-left: auto;
+        }
+
+        /* remove button */
+        &:nth-of-type(3) {
+          margin-left: 1rem;
+          color: #ce1b4c;
+        }
+      }
+    }
+
+    @media (max-width: 860px) {
+      grid-template-columns: 1fr 1fr 1fr 1fr;
+    }
+  }
+
+  .disclosure-row-test-plans {
+    display: grid;
+    row-gap: 0.5rem;
+    grid-template-columns: 2fr 2fr 1fr;
+    column-gap: 2rem;
+
+    & > :nth-of-type(3) {
+      display: block;
+    }
+
+    & > :nth-of-type(5) {
+      grid-column: span 2;
+    }
+
+    @media (max-width: 768px) {
+      grid-template-columns: 1fr;
+
+      & > :nth-of-type(3) {
+        display: none;
+      }
+
+      & > :nth-of-type(5) {
+        grid-column: initial;
+      }
+    }
+  }
+
+  .form-group-at-version {
+    display: flex;
+    flex-wrap: wrap;
+    column-gap: 1rem;
+    row-gap: 0.75rem;
+
+    select {
+      width: inherit;
+      @media (max-width: 767px) {
```

Comment:
Could be useful to have variables for breakpoints as well

Reply:
Seems to still not be possible today without some other pre/post-processor. What I have done though is to use more general breakpoints across the app.

(I've still defined those variables because this aligns with your thoughts below on potentially including postcss-loader which gets a +1 from me)